WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -101,6 +102,7 @@ def __init__( self, jobID = None, jobReport = None ):
     self.sandboxSizeLimit = gConfig.getValue( self.section + '/OutputSandboxLimit', 1024 * 1024 * 10 )
     self.cleanUpFlag = gConfig.getValue( self.section + '/CleanUpFlag', True )
     self.pilotRef = gConfig.getValue( '/LocalSite/PilotReference', 'Unknown' )
+    self.userID = gConfig.getValue( '/LocalSite/BoincUserID', '' )
```

Comment:
Would be better to call this variable self.boincUserID, nobody knows what other userIDs we can have in the future